### PR TITLE
CreateStub returns a Stub

### PR DESCRIPTION
**Pull request by VincentLanglet**
Since the implementation, `createStub` returns a Stub https://github.com/sebastianbergmann/phpunit/commit/f0d4aee11827ab6ba8a93ce7db280f4f4e280058#diff-75a3eeca974a0ce169359a31becc86004ea431fb1811f13e1ff04b06fb926aeaR1552

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Since PHP 7.1 was dropped from phpstan/phpstan, I had to remove the support of php 7.1 here @ondrejmirtes

File: src/PhpDoc/PHPUnit/MockObjectTypeNodeResolverExtension.php [hunk 1]
```diff
@@ -37,8 +37,8 @@ public function resolve(TypeNode $typeNode, NameScope $nameScope): ?Type
 		}
 
 		static $mockClassNames = [
-			'PHPUnit_Framework_MockObject_MockObject' => true,
```
Comment (ondrejmirtes) [inline src/PhpDoc/PHPUnit/MockObjectTypeNodeResolverExtension.php:40]: Why did you remove support for `PHPUnit_Framework_MockObject_MockObject`?

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: src/PhpDoc/PHPUnit/MockObjectTypeNodeResolverExtension.php (hunk 1 above)
Comment (VincentLanglet) [reply src/PhpDoc/PHPUnit/MockObjectTypeNodeResolverExtension.php:40]: it was a mistake, sorry ; it's fixed.

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you.
